docs/strategies/retry.md

```diff
@@ -97,16 +100,16 @@ new ResiliencePipelineBuilder<HttpResponseMessage>().AddRetry(optionsExtractDela
-| `Delay`            | 2 seconds                                                                  | The base delay between retries.                                                          |
-| `BackoffType`      | Constant                                                                   | The type of the back-off used to generate the retry delay.                               |
-| `UseJitter`        | False                                                                      | Allows adding jitter to retry delays.                                                    |
-| `DelayGenerator`   | `null`                                                                     | Used for generating custom delays for retries.                                           |
-| `OnRetry`          | `null`                                                                     | Action executed when retry occurs.                                                       |
-| `MaxDelay`         | `null`                                                                     | Caps the calculated retry delay to a specified maximum duration.                         |
+| Property           | Default Value                                       | Description                                                                                               |
+|--------------------|-----------------------------------------------------|-----------------------------------------------------------------------------------------------------------|
+| `ShouldHandle`     | Any exceptions except `OperationCanceledException`. | It defines a predicate to determine what results and/or exceptions are handled by the retry strategy.     |
+| `MaxRetryAttempts` | 3                                                   | The maximum number of retry attempts to use, in addition to the original call.                            |
+| `BackoffType`      | Constant                                            | The back-off algorithm type to generate the delay(s) between retry attempts.                              |
+| `Delay`            | 2 seconds                                           | The *base* delay between retry attempts. See the next section for more details.                           |
+| `MaxDelay`         | `null`                                              | If provided then the strategy caps the calculated retry delay to this value.                              |
+| `UseJitter`        | False                                               | If set to `true` a jitter (random value) is added to retry delays. See the next section for more details. |
+| `DelayGenerator`   | `null`                                              | This method allows you to **dynamically** calculate the retry delay based on runtime accessible values.   |
```

Comment:
I think I know what you mean, but "runtime accessible" doesn't seem like a clear choice of phrase here.

Reply:
How about this?
```
to **dynamically** calculate the retry delay by utilizing information that only available at runtime (like attempt number).
```

Reply:
_"by utilizing information that is only available at runtime (like the attempt number)"_